syft/serde/msgpack/serde.py

```diff
@@ -69,7 +68,7 @@
 from syft.messaging.message import SearchMessage
 from syft.messaging.message import PlanCommandMessage
 from syft.messaging.message import WorkerCommandMessage
-from syft.serde import compression
+from syft.serde import compression, msgpack
```

Comment:
```suggestion
from syft.serde import compression
from syft.serde import msgpack
```

Reply:
Any reasons for doing it this way?

Reply:
It makes the diffs of future PRs cleaner if one of those needs to change, which makes rebasing/cherry-picking/merging easier.